services/wallet/reader.go

```diff
@@ -105,6 +107,21 @@ func (r *Reader) Start() error {
 	ctx, cancel := context.WithCancel(context.Background())
 	r.cancel = cancel
 
+	go func() {
+		ticker := time.NewTicker(time.Minute)
+		defer ticker.Stop()
+		for {
+			select {
+			case <-ctx.Done():
+				return
+			case <-ticker.C:
+				r.walletFeed.Send(walletevent.Event{
+					Type: EventWalletTickCheckConnected,
```

Comment:
Where is this event handled? Cannot find another reference to it

Reply:
It is in status-desktop

Reply:
ohh sorry, missed the other PR